R/build-search-docs.R

```diff
@@ -235,7 +235,10 @@ bs4_index_data <- function(node, previous_headings, title, dir, path) {
   xml2::xml_remove(xml2::xml_find_all(node_copy, ".//*[contains(@class, 'dont-index')]"))
 
   # Helpers for XPath queries
-  all <- function(...) paste0(".//", c(...), collapse = "|")
+  all <- function(...) {
```

Comment:
This needs a comment explaining what's happening.

Reply:
added